gcp/iap-ingress/base/cloud-endpoint.yaml

```diff
@@ -1,9 +1,9 @@
 apiVersion: ctl.isla.solutions/v1
 kind: CloudEndpoint
 metadata:
-  name: $(appName)
+  name: $(ConfigMap.parameters.data.appName)
 spec:
-  project: $(project)
+  project: $(ConfigMap.parameters.data.project)
   targetIngress:
-    name: $(ingressName)
-    namespace: $(istioNamespace)
+    name: $(ConfigMap.parameters.data.ingressName)
+    namespace: $(ConfigMap.parameters.data.istioNamespace)
```

Comment:
This is fine. I left both options so that the team can understand what is possible. My project is more using the approach of a central index KfDef rather than small ConfigMap.